llvm/test/Feature/Repo/repo-opt-O0-pipeline.ll

```diff
@@ -0,0 +1,31 @@
+; RUN: opt -mtriple="x86_64-pc-linux-gnu-repo" -O0 -debug-pass=Structure < %s -o /dev/null 2>&1 | FileCheck %s
```

Comment:
It looks like these tests just list all of the passes which will be run. This could be fragile when passes are changed.

What does this test actually want to check is true?

Reply:
> 
> 
> It looks like these tests just list all of the passes which will be run. This could be fragile when passes are changed.
>
 Good point, I have removed unnecessary check.

>
> What does this test actually want to check is true?
>
These tests check the repo related passes are always run in the beginning of the ModulePass Manager for the different optimisation flags.